draft-ietf-ppm-dap.md

```diff
@@ -907,18 +1074,19 @@ abort with error "unrecognizedMessage".
 
 #### Input Share Decryption {#input-share-decryption}
 
-Each report share has a corresponding task ID, nonce, list of extensions, and encrypted
-input share. Let `task_id`, `nonce`, `extensions`, and `encrypted_input_share` denote these
-values, respectively. Given these values, an aggregator decrypts the input
-share as follows. First, the aggregator looks up the HPKE config and corresponding
-secret key indicated by `encrypted_input_share.config_id`. If not found, then it
-marks the report share as invalid with the error `hpke-unknown-config-id`.
-Otherwise, it decrypts the payload with the following procedure:
+Each report share has a corresponding task ID, report info (nonce, batch
```

Comment:
report info should be nonce, time, and extensions?

Reply:
Haha yup good catch!